template-files/latex4ei/latex4ei.sty

```diff
@@ -147,7 +147,7 @@
 	\RequirePackage[english, ngerman]{babel}
 
 	\RequirePackage{iflang}
-	\providecommand{\EngGer}[2]{\IfLanguageName{english}{#1}{#2}}
+	\providecommand{\EngGer}[2]{\IfLanguageName{english}{#2}{#1}}
```

Comment:
English comes first, german second. At least this tells the name and this is how you use it below. Why are we flipping them here?

Reply:
I know right? It felt weird to me aswell but somehow after that the languages were not flipped anymore. Because before that, when selecting `english` as the language it would become german and when selecting `ngerman` (and it has to be ngerman even though the if statement only checks "english" here) the document language turned to english. Really weird...